profiling-ffi/src/profiles.rs

```diff
@@ -23,6 +23,48 @@ pub enum SerializeResult {
     Err(Error),
 }
 
+#[repr(C)]
+pub enum UpscalingRuleAddResult {
+    // Do not use the value of Ok. This value only exists to overcome Rust -> C code generation.
+    Ok(bool),
+    Err(Error),
+}
+
+#[repr(C)]
+pub enum UpscalingInfo {
+    Poisson {
+        x: usize,
```

Comment:
`x` and `y` refer to offsets in a pprof row (which are `alloc-size` and `alloc-samples` in case of allocation profiling). I am also not happy with the naming, but I can't think of something better ATM

Reply:
I agree it's not easy to name
`x` is the offset of the value that is a kind of sum => `sum_value_offset`
`y` is the offset of the value that is a kind of count => `count_value_offset`
WDYT ?